frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/EnhancedGraphTable.java

```diff
@@ -161,6 +166,42 @@ private void subscribeToGitRepositoryFilesChanges() {
     Disposer.register(project, messageBusConnection);
   }
 
+  private void subscribeToVcsBranchChange() {
+    Topic<BranchChangeListener> topic = BranchChangeListener.VCS_BRANCH_CHANGED;
+    BranchChangeListener listener = new BranchChangeListener() {
```

Comment:
```suggestion
    val branchChangeListener = new BranchChangeListener() {
```

Reply:
+1, explicit type is of little value where the type of the right-hand side expression is obvious (like after `new`)